MeesGame/Gameplay/Content/Content.mgcb

```diff
@@ -223,7 +223,6 @@
 /processorParam:TextureFormat=Color
 /build:fog@16.png
 
-
```

Comment:
Try to avoid changing whitelines, I'm not sure but it could cause conflicts if someone else wants to change anything with content later.